client/src/components/FieldTag/index.tsx

```diff
@@ -35,17 +35,20 @@ const FieldTag: FC<FieldTagProps> = ({
     return `${split.join('-')}-tag-color`;
   };
 
+  const getFullClassName = (): string => {
+    const storedBackground =
+      hexcode === undefined ? '' : toClassName(toString(content));
+
+    return classNames('interest-team-label', storedBackground, rest.className);
```

Comment:
Is "interest-team-label" needed here?

Reply:
Yes because this contains the general styling for a FieldTag such as the border and spacing and such.

Reply:
I did rename it to be more general though. `interest-team-label` -> `field-tag-label`